python/idsse_common/idsse/common/validate_schema.py

```diff
@@ -12,9 +12,14 @@
 import os
 from typing import Optional, Union
 
-from jsonschema import FormatChecker, RefResolver  # pylint: disable=no-name-in-module
-from jsonschema.validators import validator_for
+from jsonschema import Draft202012Validator, FormatChecker
 from jsonschema.protocols import Validator
+from referencing import Registry
+from referencing.jsonschema import DRAFT202012
```

Comment:
referencing is standard python package yeah?

Reply:
As standard as json and jsonschema.

Reply:
The docs weren't super clear, but `referencing` is technically a 3rd-party library that gets installed along with jsonschema when you run `pip install jsonschema`. So we should already have it.

(Source: [pyproject.toml file from jsonschema](https://github.com/python-jsonschema/jsonschema/blob/e0f9c69a8b8299b2eba750ee5ac9340f365f8a39/pyproject.toml#L44)).

Reply:
Thanks fro the clarification. Locally I had an old jsonschema, so I directly installed referencing, and later had realized I had to update jsonschema, so I didn't know they came together, or that referencing was a third party. But this does explain why the code worked on github.